MicroProfile-Rest-Client/tck-runner/pom.xml

```diff
@@ -273,7 +273,7 @@
             <properties>
                 <!-- Rest Client Dependencies -->
                 <microprofile.rest.client.tck.version>1.0.1.payara-p2</microprofile.rest.client.tck.version>
-                <microprofile.rest.client.api.version>1.0.1.payara-p2</microprofile.rest.client.api.version>
+                <microprofile.rest.client.api.version>1.1.payara-p2</microprofile.rest.client.api.version>
```

Comment:
Payara 4 isn't on 1.1...?

Reply:
The tck version isn't but api requires 1.1 for the Async support - ran the test locally and it passes as expected